### PR TITLE
HTTP/2 chapter edits

### PR DESCRIPTION
**Pull request by rviscomi**
This is a branch of @rachellcostello's PR #334 including all of my copyedits and data viz changes.

### CODE REVIEW

**Comment by rviscomi** [comment]
@bazzadp I want to reiterate that this is an awesome chapter! I love the level of depth and passion you bring to the content. Great work!!

**Comment by bazzadp** [comment]
> @bazzadp I want to reiterate that this is an awesome chapter! I love the level of depth and passion you bring to the content. Great work!!

That kind of buttering up can only means you’ve ripped it to shreds in your editing 😀

Will take a look.

File: src/content/en/2019/http2.md [hunk 1]
```diff
@@ -165,85 +175,96 @@ Merging these two stats together, we can see the percentage of installs, of each
...
+However, the relatively low usage of fonts and images may mean that risk is not being seen as much as was feared. `<link rel="preload" ... >` tags are often used in the HTML rather than HTTP `link` headers and the meta tags are not a signal to push. Statistics in the [Resource Hints](./resource-hints) chapter show that fewer than 1% of sites use the preload HTTP `link` header, and about the same amount use preconnect which has no meaning in HTTP/2, so this would suggest this is not so much of an issue. Though there are a number of fonts and other assets being pushed, which may be a signal of this.
 
 To get around this unintended push, you can provide the `nopush` attribute in your preload header:
 
-    link: </assets/jquery.js>; rel=preload; as=script; nopush
+```
+link: </assets/jquery.js>; rel=preload; as=script; nopush
```
Comment (bazzadp) [inline src/content/en/2019/http2.md]: Is this all that's needed here? thought I had to do more as per https://github.com/HTTPArchive/almanac.httparchive.org/issues/237#issuecomment-550476367

File: src/content/en/2019/http2.md [hunk 2]
```diff
@@ -165,85 +175,96 @@ Merging these two stats together, we can see the percentage of installs, of each
...
+```
 
-It looks like 5% of preload HTTP headers do make use of this attribute, which is higher than I would have expected as I would have considered this a niche optimization. Then again, so is the use of preload HTTP headers and/or HTTP/2 push itself!
+5% of preload HTTP headers do make use of this attribute, which is higher than I would have expected as I would have considered this a niche optimization. Then again, so is the use of preload HTTP headers and/or HTTP/2 push itself!
 
 ## Issues
```
Comment (bazzadp) [inline src/content/en/2019/http2.md]: Pull back a change lost in https://github.com/HTTPArchive/almanac.httparchive.org/pull/308 to make this headingless generic and more inline with other headings.
```suggestion
## HTTP/2 Issues
```

File: src/content/en/2019/http2.md (hunk 1 above)
Comment (rviscomi) [reply src/content/en/2019/http2.md]: It _should_ be all that's needed. I can confirm that it doesn't work without the class name. But then, why aren't `code-block` styles the default for _all_ `pre` elements?

Comment (bazzadp) [reply src/content/en/2019/http2.md]: Missing a comma

https://github.com/HTTPArchive/almanac.httparchive.org/blob/f48f1eb5ab2ae8e2a2c1477d0f286dca5472016b/src/static/css/page.css#L29-L28

Should this:

```
.code-block pre {
  overflow-y: hidden;
  overflow-x: scroll;
}
```

Be this:

```
.code-block, pre {
  overflow-y: hidden;
  overflow-x: scroll;
}
```

Comment (rviscomi) [reply src/content/en/2019/http2.md]: Yes! Fixed.

**Pull request merged**